saleor/plugins/webhook/utils.py

```diff
@@ -52,14 +47,18 @@ def from_payment_app_id(app_gateway_id: str) -> Optional["PaymentAppData"]:
         try:
             app_pk = int(splitted_id[1])
         except (TypeError, ValueError):
-            return None
+            return PaymentAppData(
+                app_identifier=splitted_id[1], app_pk=None, name=splitted_id[2]
+            )
         else:
-            return PaymentAppData(app_pk, name=splitted_id[2])
+            return PaymentAppData(
+                app_pk=app_pk, app_identifier=None, name=splitted_id[2]
+            )
```

Comment:
Is there any restriction that the identifier cannot be a number?

Reply:
In the future we will add some restriction to limit identifier to domain format